packages/e2e.test.foundry-sdk-generator/generateMockOntology.js

```diff
@@ -107,10 +117,54 @@ export async function setup() {
     _: [],
     $0: "",
   });
+
+  await safeStat(testApp2Dir, "should exist");
 }
 
 export async function teardown() {
   // eslint-disable-next-line no-console
-  console.log("Test teardown: stopping API server");
+  console.log("teardown: stopping API server");
   apiServer.close();
 }
+
+await setup();
+await teardown();
+
+/**
+ * @param {string} testAppDir
+ */
+async function rmRf(testAppDir) {
```

Comment:
Split the deletion out so its more readable up above and so that the first failure doesn't cause the be ignored.